client/src/components/CohortsAlumniPage/index.js

```diff
@@ -0,0 +1,263 @@
+        description: 'Please insert correct Id',
+      });
+    }
+  }
+
+  updateWindowDimensions = () => {
+    this.setState({ width: window.innerWidth });
+  };
+
+  render() {
+    const {
+      data,
+      activeItemIndex,
+      width,
+      isCohortPages,
+      cohortName,
+    } = this.state;
+    const { type } = this.props;
+    const slidesNum =
+      width < 650
+        ? 1
+        : width < 860
+        ? 2
+        : width < 1100
+        ? 3
+        : width < 1200
+        ? 4
+        : 5;
+    const {
+      history: { push },
+    } = this.props;
+    return (
+      <UserContainer
+        rightPageColor="black"
+        headerLogo={type === 'Cohorts' ? whiteLogo : coloredLogo}
+        isCohortPages={isCohortPages}
+      >
+        {type === 'Cohorts' ? (
+          <div className="left">
+            <img src={leftSvg} alt="background" className="leftSvg" />
+          </div>
+        ) : (
+          <div>
+            <img src={leftSvg} alt="background" className="leftSvg" />
+          </div>
+        )}
+
+        <div className="right">
+          <img src={rightSvg} alt="background" className="rightSvg" />
+        </div>
+        <div className="child_container">
+          {type === 'Cohorts' ? (
+            <h1 className="title_heading">
+              <span className="title_span">Coh</span>orts
+            </h1>
+          ) : type === 'Alumni' ? (
+            <h1 className="title_heading">Alumni</h1>
+          ) : (
+            <>
+              <h1 className="title_heading">{cohortName}</h1>
+              <h2 className="cohort_alumni_heading">{cohortName} Alumni</h2>
+            </>
+          )}
+
+          <div style={{ padding: '0 80px', margin: '0 auto' }}>
+            <ItemsCarousel
+              infiniteLoop={false}
+              gutter={12}
+              activePosition="center"
+              chevronWidth={60}
+              disableSwipe={false}
+              alwaysShowChevrons={false}
+              numberOfCards={slidesNum}
+              slidesToScroll={1}
+              outsideChevron
+              showSlither={false}
+              firstAndLastGutter={false}
+              activeItemIndex={activeItemIndex}
+              requestToChangeActive={(value) =>
+                this.setState({ activeItemIndex: value })
+              }
+              rightChevron={'>'}
+              leftChevron={'<'}
+            >
+              {data.length === 0 ? (
+                <Card className="loading_card">
+                  <Skeleton loading avatar active>
+                    <Meta
+                      avatar={
+                        <Avatar src="https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png" />
+                      }
+                      title="Card title"
+                      description="This is the description"
+                    />
+                  </Skeleton>
+                </Card>
+              ) : (
+                data.map((x) => (
```

Comment:
I think this could use a more descriptive name
```suggestion
                data.map((cohort) => (
```